README.md

```diff
@@ -236,14 +245,7 @@ There exist some things you have to be careful about when using multiple functor
 
 * If you have `n` functors within one iteration and update the particle position only at the end or start of the iteration, the rebuildFrequency and the samplingRate have to be a multiple of `n`.
 
-### Inserting additional particles
-Before inserting additional particles (e.g. through a grand-canonical thermostat ),
-you always have to enforce a containerUpdate on ALL AutoPas instances, i.e.,
-on all mpi processes, by calling
-```cpp
-autoPas.updateContainerForced();
-```
-This will invalidate the internal neighbor lists and containers.
+* Functors must be marked as (not) relevant for tuning by specifying `Functor::isRelevantForTuning()`. Functors marked as relevant should have a near identical performance profile otherwise the sampling of configurations will be distorted. It is probably for the best to only mark the most expensive functor as relevant.
```

Comment:
```suggestion
* Functors can be marked as (not) relevant for tuning by specifying `Functor::isRelevantForTuning()`. Functors marked as relevant should have a near-identical performance profile otherwise the sampling of configurations will be distorted. It is recommended, to only mark the most expensive functor as relevant.
```